mods/cnc/maps/funpark01/scj01ea.lua

```diff
@@ -0,0 +1,106 @@
+RifleReinforcments = { "e1", "e1", "e1", "bike" }
+BazookaReinforcments = { "e3", "e3", "e3", "bike" }
+BikeReinforcments = { "bike" }
+
+
+ReinforceWithLandingCraft = function(units, transportStart, transportUnload, rallypoint)
+	local transport = Actor.Create("oldlst", true, { Owner = nod, Facing = 0, Location = transportStart })
+	local subcell = 0
+	Utils.Do(units, function(a)
+		transport.LoadPassenger(Actor.Create(a, false, { Owner = transport.Owner, Facing = transport.Facing, Location = transportUnload, SubCell = subcell }))
+		subcell = subcell + 1
+	end)
+
+	transport.ScriptedMove(transportUnload)
+
+	transport.CallFunc(function()
+		Utils.Do(units, function()
+			local a = transport.UnloadPassenger()
+			a.IsInWorld = true
+			a.MoveIntoWorld(transport.Location - CVec.New(0, 1))
+
+			if rallypoint ~= nil then
+				a.Move(rallypoint)
+			end
+		end)
+	end)
+
+	transport.Wait(5)
+	transport.ScriptedMove(transportStart)
+	transport.Destroy()
+
+	Media.PlaySpeechNotification(player, "Reinforce")
+end
+
+initialSong = "j1"
+PlayMusic = function()
+	Media.PlayMusic(initialSong, PlayMusic)
+	initialSong = nil
+end
+
+WorldLoaded = function()
+	nod = Player.GetPlayer("Nod")
+	dinosaur = Player.GetPlayer("Dinosaur")
+	civilian = Player.GetPlayer("Civilian")
+
+	InvestigateObj = nod.AddPrimaryObjective("Investigate the nearby village for reports of \nstrange activity")
+
+	Trigger.OnObjectiveAdded(nod, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "New " .. string.lower(p.GetObjectiveType(id)) .. " objective")
+	end)
+
+	Trigger.OnObjectiveCompleted(nod, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective completed")
+	end)
+
+	Trigger.OnObjectiveFailed(nod, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective failed")
+	end)
+
+	Trigger.OnPlayerWon(nod, function()
+		Media.PlaySpeechNotification(nod, "Win")
+	end)
+
+	Trigger.OnPlayerLost(nod, function()
+		Media.PlaySpeechNotification(nod, "Lose")
+	end)
+
+	ReachVillageObj = nod.AddPrimaryObjective("Reach the village.")
```

Comment:
I added it to make the players task more clear (he has to search for the village anyway).
Do you want me to remove it?